club/theme.json

```diff
@@ -298,9 +362,7 @@
                     }
                 },
                 "typography": {
-                    "fontSize": "var(--wp--preset--font-size--large)",
```

Comment:
Set the size to `default-title`. currently it takes h2 size which is huge.

Reply:
:+1: